guidelines/REUSE.md

```diff
@@ -133,8 +209,20 @@ _Assuming file https://my.company.com/definitions/models.json_
 ```
 
 
+### Definitions
+
+Reuse schema definitions by creating a repository of definitions.  This is done by simply hosting a file or set of files for commonly used definitions across a company or organization.
+
+Refer to [External references](#external-references) for referencing strategies.
```

Comment:
JSON References can reference within the same document so I think the _"creating a repository of definitions"_ is misleading.  In fact, Swagger already uses this for `parameters`, `responses`, `definitions`, ...

I think the point of _reuse_ is to suggest that reuse is enable through the use of JSON References.  If we need to explain more about JSON References, do it of course.